File: Obligatorio.API/Models/Agenda.cs
namespace Obligatorio.API;

public class Agenda
{
    public int id { get; set; }
    public string ci { get; set; }
    public DateOnly fechaAgenda { get; set; }

    public Agenda(int _id,
                  string _ci, 
                  DateOnly _fechaAgenda) 
    {
        this.id = _id;
        this.ci = _ci;
        this.fechaAgenda = _fechaAgenda;
    }
}

File: Obligatorio.API/models/Agenda.ts
export class Agenda {

    public nro: number;
    public ci: string;
    public fechaAgenda: Date;


    constructor(_nro: number, _ci: string, _fechaAgenda: Date) {
        this.nro = _nro
        this.ci = _ci
        this.fechaAgenda = _fechaAgenda
    }
}
